lib/rules/no-useless-escape.js

```diff
@@ -36,6 +36,8 @@ const VALID_STRING_ESCAPES = [
     "\r"
 ];
 
+const DIGITS = Array.from(Array(10).keys(), String);
```

Comment:
I have to admit, I think `const DIGITS = "0123456789".split("");` is a bit more readable.